### DISCUSSION TITLE
Service without links

### DISCUSSION THREAD

**Discussion by PhieF** [JulianPrieber/littlelink-custom#309] [Feature requests]
Hi there, thanks for your work

This might be a feature request or a question, I don't know. In your service we can add "links" to Session, Discord or other services like that, however we don't have links for those, only IDs

I was wondering if we could have an option to add IDs for example when clicking the Discord button it would open a dialog showing the ID, or display the ID with a Discord icon instead of the usual discord button (but for session it would be messy since the ID is very long)

Thanks again for your work !

**Comment by JulianPrieber** [comment]
Moved to feature request 👍

**Comment by JulianPrieber** [comment]
I've had the idea of implementing this for some time, but it's a challenge with our current system. There are several factors to consider before we can move forward with further plans. Unfortunately, I can't give a specific timeline for implementation.

The most imminent plan is the implementation of a copy button that allows you to easily copy content to your clipboard or opens a pop-up window with the option to copy.

Additionally, we aim to integrate more APIs with third-party tools, enabling buttons to open apps on different devices or generate follow requests in third-party apps and websites.

We are excited about developing and implementing new features and understand that users are eager for this as well. We are working diligently to make these plans a reality and bring new and improved experiences to our users.

**Reply by JulianPrieber** [reply]
For now, I'm doing something like **[this](https://discord.com/users/236085943541563392)** for Discord.

**Reply by PhieF** [reply]
@JulianPrieber  Hi, thanks for this answer, I guess this kind of link don't work very well, for example when I use yours I just go to my @me profile
